lib/index.js

```diff
@@ -97,11 +98,13 @@ function removeSingleArrays(obj) {
 }
 
 function readFileAsync(path, encoding) {
-  return new Promise((resolve, reject) => fs.readFile(path, {encoding}, (err, data) => {
-    if (err) {
-      reject(err);
-    } else {
-      resolve(data);
-    }
-  }));
+  return new Promise((resolve, reject) =>
```

Comment:
This function needs to use async/await as well including at line 102.

Reply:
same as above.

Reply:
See the suggestion on how it's done.